util/shop/shop.js

```diff
@@ -128,7 +128,7 @@ const royalty = (userId, callback) => {
 const buy = (userId, item, quantity, callback, errorCallback) => {
   if (utils.hasProperty(enabledPowerups, item)) {
     let { price } = enabledPowerups[item];
-    if (!Number.isNaN(Number.parseInt(quantity, 10))) {
+    if (!Number.isNaN(Number.parseInt(quantity, 10)) && enabledPowerups[item].quantified) {
```

Comment:
Add `quantified` to the previous line (`{ price, quantified }`) and just use that here

Reply:
Left this since eslint complains about quantified not being const in the `{}`